ms-internal/azure-pipelines.yml

```diff
@@ -0,0 +1,31 @@
+trigger:
```

Comment:
I believe azdo has a concept of scheduled triggers to run the request at some cadence.  We probably want to be alerted to component issues regularly rather than waiting only for PRs to trigger them.

Have you investigated that as a possibility yet?

Reply:
we haven't (yet) -- some internal docs actually say the governance scans are (soon to be) run against each-and-every push to any branch, to catch any 'severe issues' at the soonest possible time -- but I see your point of _also_ having it run on a schedule so that dependencies, etc... can be (re)validated even when we don't make changes

Reply:
hi @bderusha , just added an 'every 6 hours' scheduled governance build to the pipeline.